vowpalwabbit/json_parser/include/vw/json_parser/json_utils.h

```diff
@@ -11,7 +11,9 @@
 
```

Comment:
We should make json_utils a private header and move decision_service_interaction to a separate header that can be made public

Reply:
moved json_utils into src and left the ds struct in decision_service_utils header